docs/assignment1.md

```diff
@@ -43,21 +43,23 @@ Based on the commands we agreed to use, we developed all other functional featur
 | F6  | NPCs | Throughout the game, several NPCs will appear which can be interacted with using the `interact` command. NPCs may be friendly or hostile and could hold valuable or important items for the user to acquire.  |
 | F7  | Stats | The player has two main stats, `health` and `strength`, which they will use for combat. Naturally, `health` is an indicator of the player's remaining vitality (reaching 0 health will result in losing the game), whereas `strength` determines whether or not the player can defeat an enemy in combat, and how much damage they take during the battle.  |
-| F8  | Combat | Whenever a player engages in combat, their strength is compared to that of the enemy combatant. The outcome of the fight is determined by the difference in strength between both the player and their enemy:</br> - player strength < enemy strength: enemy survives, player loses a significant chunk of health </br> - player strength >= enemy strength: enemy defeated, player loses some health</br> - player strength >> enemy strength: enemy defeated, player loses no health |
-| F9  | Gamestate | The player may save or load the game at any moment using the `save` and `load` commands respectively. A savefile will contain all actions as performed by the player up until the point of saving, along with the appropriate initial game configuration. Whenever a savefile is loaded into the game, all actions are replayed so the user may proceed where they left off when saving. The player may quit the game after saving or loading, using the `quit` command at any time.  |
+| F8  | Combat | As mentioned in F6, an NPC may hold a precious item. To acquire this item a player may have to engage in combat with the NPC. Hence, whenever a player engages in combat using the `interact` command, their strength is compared to that of the enemy combatant. The outcome of the fight is determined by the difference in strength between both the player and their enemy:</br> - player strength / enemy strength < 1.0: enemy survives, player loses half his health plus the difference in strength </br> - player strength / enemy strength == 1.0: enemy and player both lose half of their total health</br> - player strength / enemy strength > 1.0: enemy defeated, player loses no health.</br> The result of these interactions are always shown to the gamer instantly. |
```

Comment:
The combat is ok, it leaves something to be desired though. It's not really interactive, sure some items might boost your strength. Maybe make it more interactive. In a text-based adventure game a lot is possible, you could write out entire scenes of combat and make the player choose actions. Example: The orc dodges your blow and counters with a devastating swing, what will you do? Dodge. Block. Parry. Nothing.